.github/github-token-usage.md

```
@@ -0,0 +1,59 @@
+# Using GitHub Generated Token (ghs_*)
+
+Objective: Use a GitHub-generated token (ghs_*) for automating tasks that require repository access, such as managing sandbox environments and integrating with AWS services.
+
+## Table of contents
+
+- [Using GitHub-Generated Tokens (ghs_*)](#using-github-generated-tokens-ghs)
+- [Storing the Token in AWS Secrets Manager](#storing-the-token-in-aws-secrets-manager)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)